services/winddirection.md

```diff
@@ -0,0 +1,20 @@
+# Wind direction
+
+A sensor that measures wind direction.
+
+    identifier: 0x186be92b
+    extends: _sensor
+
+## Registers
+
+    ro wind_direction: u16 ° @ reading
+
+The direction of the wind.
+
+    rw direction_offset?: i16 ° @ 0x80
+
+Offset added to direction to account for sensor calibration.
+
+    const direction_resolution?: u16 ° @ reading_resolution
```

Comment:
should it also have error?

Reply:
yeah they come in pair

Reply:
use error,. drop resolution

Reply:
> The idea of the offset would be to calibrate the direction based on the orientation of the sensor

but it's read-only, I'm not sure how would that work